manimlib/mobject/svg/svg_mobject.py

```diff
@@ -110,6 +113,16 @@ def get_mobjects_from(self, element):
 
         return result
 
+    def text_to_mobject(self, g_element):
+        from manimlib.mobject.svg.text_mobject import Text
+        font_size = self.attribute_to_float(g_element.getAttribute('font-size')) if g_element.hasAttribute('font-size') else 1.0
+        font_family = g_element.getAttribute('font-family') if g_element.hasAttribute('font-size') else ''
+        font_family_list = [s.strip() for s in font_family.split(',')]
+        content = g_element.firstChild.nodeValue
+        txt_mobject = Text(content, size=font_size*self.text_scale, font=font_family_list[0])
+        o_hight = Text('o', size=font_size*self.text_scale, font=font_family_list[0]).get_height() # vertical centering with 'o'
+        return txt_mobject.shift(UP*o_hight*0.5)
```

Comment:
I'm getting kinda lost here so does a svg text element with the value o have incorrect vertical centering does this happen for other characters. I wonder if there is a way to do this automatically and not for the 'o' degenerate case

Reply:
By default Text mobject is centered about the top of small letters (red dot below). I think the correct behavior is to use the central line of small letters (green dot).

![centering](https://user-images.githubusercontent.com/11081197/82220456-54944980-9927-11ea-9a2e-194a3c13a561.png)

Without a half-o height shift, the result in PR looks like this (which seems to be incorrect).

![shifted_center](https://user-images.githubusercontent.com/11081197/82220555-78578f80-9927-11ea-8778-7c84792323a3.png)

Also, I've looked at `TextWithFixHeight` class in `text_mobject.py` for reference of text height measuring.

Reply:
`TextWithFixHeight` will create problems thats why i removed that in https://github.com/3b1b/manim/pull/1071